pkg/metrics/metrics.jsx

```diff
@@ -452,10 +453,14 @@ class CurrentMetrics extends React.Component {
         if (this.state.cpuCoresUsed.length > 1) {
             const top_cores = this.state.cpuCoresUsed.map((v, i) => [i, v]).sort((a, b) => b[1] - a[1])
                     .slice(0, 16);
-            cores = top_cores.map(c => <Flex key={c[0]} justifyContent={{ default: 'justifyContentSpaceBetween' }}>
-                <FlexItem>{ cockpit.format(_("Core $0"), c[0]) }</FlexItem>
-                <FlexItem>{c[1]}%</FlexItem></Flex>
-            );
+            cores = (<Grid className='cpu-all' component='dl'>
+                {top_cores.sort((a, b) => a[0] - b[0]).map(c =>
+                    <React.Fragment key={c[0]}>
+                        <GridItem component='dt'>{ cockpit.format(_("Core $0"), c[0]) }</GridItem>
+                        <GridItem component='dd'>{c[1]}%</GridItem>
+                    </React.Fragment>)
+                }
+            </Grid>);
```

Comment:
The structure looks fine to me. However, the .sort()ing needs to go. `top_cores` already got sorted by descending usage two lines above, and it's been like that since day one of the popup. If we want to change that, I propose to do that in a separate PR (or at least commit).